flink-cdc-common/src/main/java/com/ververica/cdc/common/schema/CollectionFilters.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2023 Ververica Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ververica.cdc.common.schema;
+
+import com.ververica.cdc.common.event.TableId;
+
+/**
+ * Provides factory methods for obtaining {@link CollectionFilter} instances as per the current
+ * connector configuration.
+ */
+public interface CollectionFilters {
+
+    CollectionFilter collectionFilter();
+
+    /** A filter for data collections. */
+    @FunctionalInterface
+    interface CollectionFilter {
+        boolean isIncluded(TableId id);
+    }
+}
```

Comment:
I think this interface may be redundant at the moment.  
`TableFilter` can meet the requirements.
If there are more complex scenarios, we can consider introducing it again.